### DISCUSSION TITLE
[Disregard] https://plausible.io/api/event is returning 404

### DISCUSSION THREAD

**Discussion by vedovelli** [plausible/analytics#1606] [Anything Else]
Hello everybody.

I've got that URL from the docs, in this page https://plausible.io/docs/proxy/guides/netlify

Proxy is working fine for https://plausible.io/js/plausible.outbound-links.js but not for https://plausible.io/api/event.

Best regards,

Vedovelli

**Comment by vedovelli** [comment]
Never mind. It's a non-issue.

**Reply by metmarkosaric** [reply]
thanks for the update @vedovelli!

**Comment by sandstrom** [comment]
@metmarkosaric If possible, close all old discussions. Less noise and easier to search discussions.
